cid/helpers/quicksight/__init__.py

```diff
@@ -238,6 +238,28 @@ def discover_dashboard(self, dashboardId: str) -> Dashboard:
             templateAccountId = _definition.get('sourceAccountId')
             templateId = _definition.get('templateId')
             region = _definition.get('region', 'us-east-1')
+
+            # Checking for version override in template definition 
+                 
+            version = dashboard.deployedTemplate.version
+            
+            version_obj = _definition.get('versions')
+            
+            if not version_obj: raise CidCritical("No `versions` key find in dashboard definition resources.yml")
```

Comment:
This will break update of other dashboards, not managed by CID bulidin

Reply:
Resolved in [commit](https://github.com/aws-samples/aws-cudos-framework-deployment/pull/445/commits/0844d19fbecdc35ad56982a5040768732f854e1a)